app/experimenter/experiments/constants/nimbus.py

```diff
@@ -292,6 +292,19 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_NO_ENTERPRISE_OR_RECENT_VPN = NimbusTargetingConfig(
+    name="No enterprise and no VPN connection in the last 30 days",
+    slug="no_enterprise_and_no_vpn_connection_last_30_days",
+    description="Exclude enterprise & users who have used MozVPN in the last 30 days",
+    targeting=(
+        f"{TARGETING_NO_ENTERPRISE.targeting} && "
+        '(("e6eb0d1e856335fc" in attachedFxAOAuthClients|mapToProperty("id")) ? '
+        '(attachedFxAOAuthClients[.id == "e6eb0d1e856335fc"].lastAccessedDaysAgo > 29) : '
```

Comment:
Yowza 😲

Reply:
Yeah, good times, right?  If JEXL's `||` short-circuited, like JavaScript's does, I wouldn't have had to do this.  C'est la vie.

Reply:
Indeed, the integration testing is upset about something.  I'll go re-test in the ASRouter devtools..

Reply:
Hmph, works in devtools.

Reply:
Amusingly, the failure here was because the targeting slug name was long enough that, when concatenated with the stuff added by the integration testing infrastructure made the experiment name longer than 80 chars, which caused experimenter to just stop.

Might be worth adding a lint for this, as it would be a lot less work to debug if someone else stumbles against this.

Reply:
Oh interesting!!  Good catch 👍

Reply:
Filed https://github.com/mozilla/experimenter/issues/7113 as a followup